test/extended/operators/operators.go

```diff
@@ -242,6 +242,7 @@ func surprisingConditions(co objx.Map) ([]configv1.ClusterOperatorStatusConditio
 			if cond.Get("status").String() != string(expected) {
 				reason := cond.Get("reason").String()
 				message := cond.Get("message").String()
+				status := cond.Get("status").String()
```

Comment:
nit: worth moving this up in front of the `if` so we don't have to get it twice?  And then we could also use that local `status` variable when appending to `badConditions` below, instead of getting it out a third time.